src/helpers/pivot/pivot_helpers.ts

```diff
@@ -209,6 +209,9 @@ export function toNormalizedPivotValue(
     typeof groupValue === "boolean"
       ? toString(groupValue).toLocaleLowerCase()
       : toString(groupValue);
+  if (groupValueString === "null") {
```

Comment:
That should never happen because of the `if` above, no?

Reply:
It can be a FunctionResultObject :/